src/omv/ports/stm32/main.c

```diff
@@ -489,6 +489,9 @@ int main(void)
 
     // Initialise low-level sub-systems. Here we need to do the very basic
     // things like zeroing out memory and resetting any of the sub-systems.
+    py_lcd_init0();
+    py_fir_init0();
+    py_tv_init0();
```

Comment:
Why is this important ?

Reply:
These modules deinit all resources when they run. Which include the cambus. So, this has to happen before sensor_init0().